src/zenml/steps/utils.py

```diff
@@ -112,6 +124,20 @@ def parse_return_type_annotations(
     Returns:
         - A dictionary mapping output names to their output signatures.
     """
+
+    def _define_output_name(
+        artifact_config: Optional["ArtifactConfig"],
+        i: int,
+        original_output_names: Optional[List[str]] = original_output_names,
+    ) -> Optional[str]:
+        output_name: Optional[str] = None
```

Comment:
Is this still necessary? `artifact_config.name` will always be the `original name`, correct?